backend/open_webui/migrations/versions/f1e2d3c4b5a6_add_access_grant_table.py

```
@@ -0,0 +1,369 @@
+
+    now = int(time.time())
+    inserted = set()
+
+    for table_name, resource_type in resource_tables:
+        if table_name not in existing_tables:
+            continue
+
+        # Query all rows
+        try:
+            result = conn.execute(
+                sa.text(f'SELECT id, access_control FROM "{table_name}"')
```

Ruff (0.15.0)

[error] Possible SQL injection vector through string-based query construction

(S608)